modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -0,0 +1,28 @@
+= Install Redpanda for FIPS compliance
+
+To run Redpanda in FIPS compliance mode, you must install the following additional packages after installing Redpanda:
+
+`redpanda-fips`
+
+- Contains the OpenSSL FIPS-approved module and scripts required to set up and run Redpanda in FIPS-compliance mode.
+- Depends upon the successful installation of the `redpanda` package.
+- Includes the `fips.so` cryptographic provider (built from OpenSSL v3.0.9, which is the latest FIPS 140-2 approved module) and a copy of the OpenSSL application.
+- Executes `openssl fipsinstall` against the `fips.so` module, which generates a `fipsmodule.cnf` file that is used during the module’s POST (power-on-self-test) to validate the integrity of the module.
+
+`redpanda-rpk-fips`
+
+- Contains a version of `rpk` built with the https://github.com/microsoft/go[Microsoft GoLang compiler^] and https://github.com/microsoft/go-crypto-openssl[Microsoft’s Go Crypto OpenSSL package^] to which `rpk` is linked, and uses the FIPS-approved version of OpenSSL.
+
+To install Redpanda for FIPS compliance, run:
+
+[,bash]
+----
+sudo apt install -y redpanda-rpk-fips redpanda-fips
```

Comment:
since this is beta, we should add guides on how to use the unstable repository for apt for this installation

Reply:
^^ @michael-redpanda What should we be telling beta users wrt executing the commands shown the doc (how are these commands impacted by beta installs/repo?)

Reply:
No difference irt beta (except for using the beta repo as @Deflaimun pointed out)

Reply:
@Feediver1 we have the install beta doc page here : https://github.com/redpanda-data/docs/blob/v-WIP/24.2/modules/get-started/pages/install-beta.adoc?plain=1#L109

We could either:
- link to this page
- create a partial and include here
- create a conditional doc (if beta do this, else do that). So after the beta period has expired this section wouldn't appear.

Reply:
@michael-redpanda are FIPS packages only available via apt? Are we going to have docker images too?

Reply:
@Deflaimun Docker images and Kubernetes support are on the roadmap but are not available at this moment. Right now the sole methods of installation are the binary or rpm/deb packages.

Reply:
@Deflaimun Yes (apt and yum/dnf).  I believe they should be availabler currently in nightly